web-components/package.json

```diff
@@ -50,7 +50,7 @@
     "@momentum-ui/tokens": "1.7.1",
     "@momentum-ui/utils": "6.2.15",
     "lit-element": "2.3.1",
-    "lit-html": "1.2.1"
+    "lit-html": "^1.2.1"
   },
   "dependencies": {
     "@interactjs/actions": "1.10.3",
```

Comment:
can you add node version restriction here? something like this.

```
"engines": {
    "node": ">=20.0.0"
  },
  ```

Reply:
Not required.